### PR TITLE
Kyles feature

### LINKED ISSUE #2: Kyles issue to resolve

### PR DESCRIPTION
**Pull request by Kyle-LeMarbe**
This is kyles pull request to get my feature into main.

### CODE REVIEW

File: codefile.cs [hunk 1]
```diff
@@ -9,4 +9,6 @@ void mytestfunctionCheckDirectlyIntoBase()
 void anotherfunctiondirectlytomaster()
 {
   throw Exception();
-}
\ No newline at end of file
+}
+
+//another test with comment  I shoulnt be able to commit this....
```
Comment (Kyle-LeMarbe) [inline codefile.cs]: my checkin note here.

**Review by Kyle-LeMarbe — COMMENTED** [review]
I can review my own code but cannot approve it on my own.  Great!

File: codefile.cs (hunk 1 above)
Comment (dBManiac-) [reply codefile.cs]: Looks good.

**Review by dBManiac- — APPROVED** [review]
Approved

**Pull request merged**